src/views/main-view.js

```diff
@@ -80,11 +80,14 @@ MainView.prototype._initialize = function () {
 
   this.model.on('changeActivePaymentView', function (id) {
     if (id === PaymentMethodsView.ID) {
-      classlist.add(paymentMethodsViews.element, 'braintree-methods--active');
+      classlist.add(paymentMethodsViews.container, 'braintree-methods--active');
+      classlist.remove(paymentMethodsViews.container, 'braintree-methods-container--inactive');
```

Comment:
It doesn't look like we are using this property anymore. We removed it and rechecked that it didn't effect anything.

If you have an idea on why we should keep this let us know!